load-code/pe/parse-pe-header.yml

```diff
@@ -24,11 +24,11 @@ rule:
             - number: 0x4D
             - number: 0x5A
       - and:
-        # - offset: 0x3C = IMAGE_DOS_HEADER.e_lfanew
+        - offset: 0x3C = IMAGE_DOS_HEADER.e_lfanew
```

Comment:
less FPs but with the potential to miss where this happens in another function, for example